rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -276,3 +354,25 @@ pub fn livesplit_set_text_color(color: Color) {
     let mut state = LIVESPLIT_STATE.lock().unwrap();
     state.layout.general_settings_mut().text_color = color;
 }
+#[rebo::function("LiveSplit::get_total_playtime")]
+pub fn livesplit_get_total_playtime() -> String {
+    LiveSplitTime::format(TotalPlaytime::get_total_playtime_as_timespan())
+}
+#[rebo::function("LiveSplit::set_digits_format_and_accuracy")]
+pub fn livesplit_set_digits_format_and_accuracy(digits_format: DigitsFormat, accuracy: Accuracy) {
+    let digits = match digits_format {
+        DigitsFormat::SingleDigitSeconds => LiveSplitDigitsFormat::SingleDigitSeconds,
+        DigitsFormat::DoubleDigitSeconds => LiveSplitDigitsFormat::DoubleDigitSeconds,
+        DigitsFormat::SingleDigitMinutes => LiveSplitDigitsFormat::SingleDigitMinutes,
+        DigitsFormat::DoubleDigitMinutes => LiveSplitDigitsFormat::DoubleDigitMinutes,
+        DigitsFormat::SingleDigitHours => LiveSplitDigitsFormat::SingleDigitHours,
+        DigitsFormat::DoubleDigitHours => LiveSplitDigitsFormat::DoubleDigitHours,
+    };
+    let accuracy = match accuracy {
+        Accuracy::Seconds => LiveSplitAccuracy::Seconds,
+        Accuracy::Tenths => LiveSplitAccuracy::Tenths,
+        Accuracy::Hundredths => LiveSplitAccuracy::Hundredths,
+        Accuracy::Milliseconds => LiveSplitAccuracy::Milliseconds,
+    };
```

Comment:
These should rather be From-impls, e.g. `impl From<DigitsFormat> for LiveSplitDigitsFormat`.